charts/images.yaml

```diff
@@ -8,7 +8,7 @@ images:
 - name: etcd
   sourceRepository: github.com/gardener/etcd-custom-image
   repository: eu.gcr.io/gardener-project/gardener/etcd
-  tag: "v3.4.13-bootstrap-12"
+  tag: "v3.4.26-2"
```

Comment:
why we are not following the same naming convention
```
v3.4.26-bootstrap-x
```

Reply:
The reason we initially used `bootstrap` suffix was because there was already a `v3.4.13` image, and we wanted clear differentiation between the old image and the new bootstrap-script-included image. Hence the suffix. It was then a conscious decision to remove the bootstrap suffix from later images, since the bootstrap script as part of the etcd-custom-image was by default for v3.4.14+ images.